### PR TITLE
Use gradle-embulk-plugins

### PR DESCRIPTION
**Pull request by hiroyuki-sato**
Hello, @takumakanari 

Could you take a look at this PR when you get a chance?
This PR doesn't add/change features.

I'll plan to release 0.3.2 after this PR merged.

* Use [gradle-embulk-plugins](https://github.com/embulk/gradle-embulk-plugins).
* Update Jsonpath to 2.4.0
* Add GitHub Action CI
* Fix deprecated warning
* Update gradle to 6.2

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -66,50 +51,58 @@ task checkstyle(type: Checkstyle) {
...
+gem {
+    authors = [ "Hiroyuki Sato","Takuma kanari" ]
+    email = [ "hiroysato@gmail.com","chemtrails.t@gmail.com"]
+    // "description" of the gem is copied from "description" of your Gradle project.
+    summary = "JSON parser with JSONPath plugin for Embulk"
+    homepage = "https://github.com/hiroyuki-sato/embulk-parser-jsonpath"
+    licenses = [ "MIT" ]
+
+    generateRubyCode = false  // Avoiding generate lib/embulk/parser/jsonpath.rb
+    into("lib/embulk/parser/") {
```
Comment (takumakanari) [inline build.gradle:82]: Do we still need this section?

Comment (hiroyuki-sato) [reply build.gradle:82]: Thank you for your comment. I'll take a look more deeply. I don't understand completely yet.

Comment (hiroyuki-sato) [reply build.gradle:82]: Let's double-check later. I still need this part.
embulk-parser-jsonpath supports guess feature. gradle plugins doesn't support guess yet. So, I copy these codes explicitly.
https://gist.github.com/hiroyuki-sato/3e261ef8eb2d092b3d83ae637b9ffd29#gistcomment-3175408

Comment (takumakanari) [reply build.gradle:82]: I see.

**Review by takumakanari — APPROVED** [review]
LGTM

**Comment by hiroyuki-sato** [comment]
Thanks!

**Pull request merged**